src/Control/Monad/Aff.js

```diff
@@ -96,6 +96,26 @@ exports._forkAff = function (nonCanceler, aff) {
   };
 }
 
+exports._forkAll = function (nonCanceler, foldl, affs) {
+  var voidF = function(){};
+
+  return function(success, error) {
+    foldl(function(_) {
+      return function(aff) {
+        aff(voidF, voidF);
+      };
+    })({})(affs);
+
+    try {
+      success({});
+    } catch(e) {
+      error(e);
+    }
+
+    return nonCanceler;
```

Comment:
Ok, so to make sure I'm getting it right: Collect all the cancellers in the fold, then return a canceller that propagates to all those cancellers. Should I also push that array of cancellers to `success`?